### ISSUE TITLE
Support for dataframe import from HDF5 files

### ISSUE THREAD

**Issue by fkromer** [dmnfarrell/pandastable#178]
I could not get any hits when searching for "hdf5" on https://pandastable.readthedocs.io . Are you supporting imports of pandas dataframes from HDF5 files already?

**Comment by dmnfarrell** [comment]
Yes I'm familiar with hdf5. I used to use messagepack for persisting project files but pandas dropped support for it. I can add hdf5 support quite easily I should think.  Are you talking about dataexplore or the pandastable library.

**Comment by fkromer** [comment]
Cause I need a visualization solution for my coworkers I'm talking about dataexplorer. I'd be able to PR as well. However as I'm not able to implement and PR the next time it would be great if you could add support for it.

**Comment by dmnfarrell** [comment]
Ok I'll take a look.

**Comment by fkromer** [comment]
Great, thanks a lot!

**Comment by dmnfarrell** [comment]
Have added basic import to file menu. See if it works. You'll need the latest version from the repository though.

**Comment by fkromer** [comment]
I gave `dataexplore` a short try on Ubuntu running Python 3.8.2 (`sudo apt-get install python3-tk`, `~/github/pandastable$ pip3 install --user -e .`, `~/github/pandastable$ dataexplore`) with the probably [most simple hdf5 file](http://docs.h5py.org/en/stable/quick.html#appendix-creating-a-file) possible:
```
$ python3
Python 3.8.2 (default, Jul 16 2020, 14:00:26) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import h5py
>>> import numpy as np
>>> with h5py.File("mytestfile.hdf5", "w") as f:
>>>    dset = f.create_dataset("mydataset", (100,), dtype='i')
```

`dataexplore` did not complain :+1:
